### PR TITLE
Additional to fix #12104 for CoreRT issue #3672 (armel tizen)

### PR DESCRIPTION
**Pull request by sergign60**

### CODE REVIEW

**Comment by sergign60** [comment]
CC: @jkotas @Dmitri-Botcharnikov @BredPet @mskvortsov

**Comment by sergign60** [comment]
@dotnet-bot Tizen armel Cross Debug Build test

**Comment by sergign60** [comment]
@dotnet-bot test Tizen armel Cross Debug Build

**Comment by sergign60** [comment]
@dotnet-bot Tizen armel Cross Debug Build test

**Comment by sergign60** [comment]
@dotnet-bot test Tizen armel Cross Debug Build

**Comment by sergign60** [comment]
@jkotas please review fix in regalloc

File: src/jit/regalloc.cpp [hunk 1]
```diff
@@ -6352,7 +6352,7 @@ void Compiler::rpPredictRegUse()
 
         // if there are PInvoke calls and compLvFrameListRoot is enregistered,
         // it must not be in a register trashed by the callee
-        if (info.compCallUnmanaged != 0)
+        if (info.compCallUnmanaged != 0 && (info.compLvFrameListRoot != BAD_VAR_NUM))
```
Comment (jkotas) [inline src/jit/regalloc.cpp]: Would it be better to just check for `if (info.compLvFrameListRoot != BAD_VAR_NUM)` here?

**Pull request merged**